scripts/docker/downloadEnvironmentVariables.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+# Fetch environment from LastPass, write to ${ENV_DEST}/${DEPLOYMENT_NAME}/.env
+# Is modified from ../../scripts/bash/downloadEnvironmentVariables.sh
```

Comment:
Minor: downloadEnvironmentVariables.sh will be deleted once we swap out to this new infra. So not a biggie.